contracts/test/integration/utils/AllRoles.sol

```diff
@@ -71,8 +69,6 @@ contract AllRoles is RoleTesting, AllRolesConfig {
         allAddresses[4].push(ArbitrumAddresses.EOA_2);
         allAddresses[4].push(ArbitrumAddresses.EOA_3);
 
-        allAddresses[5].push(ArbitrumAddresses.PCV_GUARD_ADMIN);
```

Comment:
this shouldn't be removed. pcv guard admin role is still given to pcv guard admin